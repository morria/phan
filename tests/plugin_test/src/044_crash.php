<?php
// Phan should not crash
class example {
    private function validFunction() : void {
    };
    private function $notAFunction = function() : void {
    };
}
