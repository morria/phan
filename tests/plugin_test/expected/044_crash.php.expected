src/044_crash.php:5 PhanSyntaxError Fallback parser diagnostic error: '}' expected.
